supervisor/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/LeonidVas/tarantool-supervisor/supervisor/api/supervisorhttp"
+	"github.com/LeonidVas/tarantool-supervisor/supervisor/core"
+)
+
+// args describes the parsed arguments.
+type args struct {
+	//CfgPath - path to Supervisor config.
+	CfgPath string
+	// Addr - address to start the HTTP server(host:port).
+	Addr string
+}
+
+// parseArgs returns the parsed arguments.
+func parseArgs() *args {
+	var args args
+	flag.StringVar(&args.CfgPath, "cfg", "cfg.json",
+		"path to Supervisor config.")
+	flag.StringVar(&args.Addr, "addr", "127.0.0.1:8080",
+		"address to start the HTTP server(host:port).")
+	flag.Parse()
+
+	return &args
+}
+
+// parseCfg parse the Supervisor JSON config.
+func parseCfg(path string) (*core.Cfg, error) {
+	// Check is the file exists.
+	if _, err := os.Stat(path); err != nil {
+		return nil, err
+	}
```

Comment:
```go
if _, err := os.Stat(path); os.IsNotExists(err) {
    return nil, fmt.Errorf("Specified config file %s doesn't exists", path)
} else if err != nil {
    return nil, fmt.Errorf("Specified config file can't be used: %s", err)
}
```

Reply:
This clarification doesn`t need, all the necessary information contained in the error.
`no such file or directory`